### ISSUE TITLE
Is the Badgy supposed to charge the battery?

### ISSUE THREAD

**Issue by vidax** [sqfmi/badgy#14]
I have the LIR2450 loaded in the badgy, and when I plug the USB, I see a red light in the back. 
I am wondering if the badgy is able to charge the battery because when I remove the USB after a few hours, the badgy does not seem to work on battery.

**Comment by sqfmi** [comment]
Hi @vidax,

The USB should recharge the LIR2450 battery. Please ensure you have the battery inserted in the correct polarity, and the battery has been sufficiently charged (3.7V - 4.2V).

**Comment by chunkysteveo** [comment]
Ahh crap....! ...and with that I released some magic smoke from the MCP73831 charge IC! 👎  Was probing around from the USB V+ and touched the VDD of the chip and it went "pooof" (was plugged in to USB at the time). I have some spare MCP73831 circuits, I may try to de-solder and re-apply one.

While poking about looking at the schematic and the PCB layout - I did notice that the LED D4 which is the charge state LED has a current limiting resistor of 2K Ohm. Should this not be more in line with common LED resistors, and the spec sheet, around 200-470Ohm? Or is this high to work with the High-Impedance state of the STAT line as well as Hi and Low?
